site/en/guide/autodiff.ipynb

```diff
@@ -947,11 +947,11 @@
       "source": [
         "Some `tf.Operation`s are **registered as being non-differentiable** and will return `None`. Others have **no gradient registered**.\n",
         "\n",
-        "The [tf.raw_ops](https://www.tensorflow.org/api_docs/python/tf/raw_ops) page shows which low-level ops have gradients registered.\n",
+        "The `tf.raw_ops` page shows which low-level ops have gradients registered.\n",
         "\n",
         "If you attempt to take a gradient through a float op that has no gradient registered the tape will throw an error instead of silently returning `None`. This way you know something has gone wrong.\n",
         "\n",
-        "For example the `tf.image.adjust_contrast` function wraps [raw_ops.AdjustContrastv2](https://www.tensorflow.org/api_docs/python/tf/raw_ops#.AdjustContrastv2) which could have a gradient but the gradient is not implemented:\n"
```

Comment:
Same case...

Reply:
Hi, same as in https://github.com/tensorflow/docs/pull/1736#discussion_r518244265 - I thought these get auto-linked to API references during the notebook-to-TensorFlow.org preprocessing